### PR TITLE
Fix #51: Ripples are almost invisible

### LINKED ISSUE #51: Ripples are almost invisible if they are put inside the drawer-panel
If you put `paper-checkbox` or `paper-radio-button` inside the `paper-drawer-panel` you will notice if you tap on them the ripple is very light, almost invisible.

```
<paper-drawer-panel>
  <div drawer></div>
  <div main>
    <div>
      <paper-button paper-drawer-toggle raised>toggle drawer</paper-button>
    </div>
    <div style="padding: 30px;">
      <paper-checkbox></paper-checkbox>
    </div>
  </div>
</paper-drawer-panel>
```

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: paper-drawer-panel.css [hunk 1]
```diff
@@ -23,7 +23,19 @@ iron-selector > #drawer {
...
+  * An issue was found when `will-change: transform` is used for #drawer
+  * It seems to be a stacking context issue that reduces paper-ripple's 
+  * opacities when the drawer is visible.
+  * The problem is solved when `style.opacity < 1` or when the background color
+  * has an alpha channel < 1
+  *
+  *
+  * Fixes:
+  * PolymerElements/paper-drawer-panel/issues/51 and
+  * Polymer/polymer-element-catalog/issues/66
```
Comment (frankiefu) [inline paper-drawer-panel.css]: We need to file an issue on Chrome and put the tracking # here.

**Comment by blasten** [comment]
@frankiefu The issue is open on https://code.google.com/p/chromium/issues/detail?id=501135

**Comment by blasten** [comment]
PTAL @frankiefu. now, it just works!

``` css
.narrow-layout > #drawer {
  will-change: transform;
}
```

😀

**Comment by frankiefu** [comment]
LGTM :+1:

**Pull request merged**